components/sections/navigation/DesktopNavigation.tsx

```diff
@@ -5,7 +5,7 @@ import { useRouter } from 'next/router'
 
 // Components
 import { Button } from '../../common'
-import { Menu } from "@headlessui/react";
+import { Menu } from '@headlessui/react'
```

Comment:
Maybe not blocking, but this changes a number of `"` double quotes over to `'` single quotes: we can push this pr through but would probably be good to get the eslint/prettier checks in here along with a `package.json` script to easy execute linting

Reply:
It looks like prettier was kicking in. Not sure why it's so different though.

Reply:
We can move this forward: maybe we re-visit linting/formatting in the future 👀